docs/openapi.yaml

```diff
@@ -2214,13 +2251,13 @@ paths:
                 type: string
               example: ERROR - 2018-10-08T20:15:11Z - The requested object DbId is
                 not found
-  /topography-options:
+  /topographies:
```

Comment:
Why did these location information endpoints change, maybe a merging or out of date issue? They don't match the controller specified endpoints?

Reply:
I didn't intentionally changes these, so probably one of the issues you mentioned, or a copy and paste issue! I'll look into it.

Reply:
Changed and pushed